app/util/env.ts

```diff
@@ -0,0 +1,4 @@
+const NODE_ENV = 'NODE_ENV';
+
+// eslint-disable-next-line import/prefer-default-export
+export const isDevelopment = () => process.env[NODE_ENV] === 'development';
```

Comment:
S: we may not realy need a const on this as it's only used once and in s very specific scope (envs).
I suggest the following:
```suggestion
export const isDevelopment = () => process.env['NODE_ENV'] === 'development';
```
Also prefer default export is not a rule anymore.